AWS/DirectoryInsights/get-jcdirectoryinsights.py

```diff
@@ -0,0 +1,76 @@
+import requests, datetime, json, boto3, os, gzip
+
+def jc_directoryinsights(event, context):
+    jcapikey = os.environ['JCAPIKEY']
+    incrementType = os.environ['incrementType']
+    incrementAmount = int(os.environ['incrementAmount'])
+    bucketName = os.environ['BucketName']
+
+    now = datetime.datetime.utcnow()
+
+    if incrementType == "minutes":
+        start_dt = now - datetime.timedelta(minutes=incrementAmount)
+    elif incrementType == "minute":
+        start_dt = now - datetime.timedelta(minutes=incrementAmount)
+    elif incrementType == "hours":
+        start_dt = now - datetime.timedelta(hours=incrementAmount)
+    elif incrementType == "hour":
+        start_dt = now - datetime.timedelta(minutes=incrementAmount)
+    elif incrementType == "days":
+        start_dt = now - datetime.timedelta(days=incrementAmount)
+    elif incrementType == "day":
+        start_dt = now - datetime.timedelta(days=incrementAmount)
+    else:
+        raise Exception("Unknown increment value.")
+        return
```

Comment:
This return will never be executed due to the raise above it, you can just nuke it